src/ObjectInfo/DoctrineORMInfo.php

```diff
@@ -62,19 +62,19 @@ private function getAssocsConfig(ClassMetadata $metadata, array $assocNames): ar
         $assocsConfigs = [];
 
         foreach ($assocNames as $assocName) {
-            if (!$metadata->isAssociationInverseSide($assocName)) {
+            $associationMapping = $metadata->getAssociationMapping($assocName);
+
+            if (isset($associationMapping['inversedBy'])) {
                 continue;
             }
 
             $class = $metadata->getAssociationTargetClass($assocName);
 
             if ($metadata->isSingleValuedAssociation($assocName)) {
-                $nullable = ($metadata instanceof ClassMetadataInfo) && isset($metadata->discriminatorColumn['nullable']) && $metadata->discriminatorColumn['nullable'];
```

Comment:
I didn't understood this line `discriminatorColumn` is for inheritance, not sure what is the relation with the nullable thing.